### PR TITLE
Fix format-security errors

### PR DESCRIPTION
**Pull request by tu-maurice**
A few format-security errors I've run into.

### CODE REVIEW

File: src/emu/debugger/src/elements/imgui_debugger_prefs.cpp [hunk 1]
```diff
@@ -54,7 +54,7 @@ namespace eka2l1 {
 
         const std::string background_str = common::get_localised_string(localised_strings, "pref_personalize_background_string");
 
-        ImGui::Text(background_str.c_str());
+        ImGui::Text("%s",background_str.c_str());
```
Comment (pent0) [inline src/emu/debugger/src/elements/imgui_debugger_prefs.cpp:57]: Can you ask space after the comma? It's a codestyle which you can see around here.

File: src/emu/debugger/src/elements/imgui_debugger_user_utils.cpp [hunk 2]
```diff
@@ -115,7 +115,7 @@ namespace eka2l1 {
             const std::string attached_msg = common::get_localised_string(localised_strings, "error_dialog_message");
             const std::string ok_str = common::get_localised_string(localised_strings, "ok");
 
-            ImGui::Text(attached_msg.c_str());
+            ImGui::Text("%s",attached_msg.c_str());
```
Comment (pent0) [inline src/emu/debugger/src/elements/imgui_debugger_user_utils.cpp:118]: ditto

File: src/emu/debugger/src/imgui_debugger.cpp [hunk 3]
```diff
@@ -398,7 +398,7 @@ namespace eka2l1 {
                         ImGui::PopStyleVar();
 
                         const std::string unavail_tt = common::get_localised_string(localised_strings, "file_menu_packages_item_unavail_msg");
-                        ImGui::SetTooltip(unavail_tt.c_str());
+                        ImGui::SetTooltip("%s",unavail_tt.c_str());
```
Comment (pent0) [inline src/emu/debugger/src/imgui_debugger.cpp:401]: ditto

File: src/emu/debugger/src/elements/imgui_debugger_install_device.cpp [hunk 4]
```diff
@@ -81,7 +81,7 @@ namespace eka2l1 {
                 const std::string install_device_welcome_msg = common::get_localised_string(localised_strings,
                     "install_device_wizard_welcome_msg");
 
-                ImGui::TextWrapped(install_device_welcome_msg.c_str());
+                ImGui::TextWrapped("%s",install_device_welcome_msg.c_str());
```
Comment (pent0) [inline src/emu/debugger/src/elements/imgui_debugger_install_device.cpp:84]: This is intentional for now sorry... I think you have to lower down this one level if it's an error. Currently ImGui does not detect newline from the final formatted string but only in the type format string (first argument), so we have to do this :(

**Review by pent0 — CHANGES_REQUESTED** [review]
In addtion please rename the commit to describe the section you are modifying and the purpose of the commit.

> debugger: Fix format-security errors

File: src/emu/debugger/src/elements/imgui_debugger_install_device.cpp (hunk 4 above)
Comment (tu-maurice) [reply src/emu/debugger/src/elements/imgui_debugger_install_device.cpp]: I added -Wno-format-security for that file, that seems to fix my build error as well.

**Comment by pent0** [comment]
Thanks for the additional cmake flags

**Review by pent0 — APPROVED** [review]

**Pull request merged**